lib/toptranslation_cli/pull.rb

```diff
@@ -48,6 +60,27 @@ def project_locales
       def project
         @project ||= ToptranslationCli.connection.projects.find(ToptranslationCli.configuration.project_identifier)
       end
+
+      def files_to_download
+        project&.documents&.flat_map do |document|
+          document.translations.map do |translation|
+            {
+              path: path(document, translation.locale),
+              document: document,
+              sha1: translation.sha1,
+              locale: translation.locale
+            }
+          end
+        end
+      end
+
+      def download(file, bar)
+        file[:document].download(file[:locale].code, path: file[:path], file_type: 'yaml') do |n, total|
```

Comment:
Why does it say `yaml` there? Did we fix the filetype somewhen for debugging?

Reply:
Good point. It was yaml before, so I kept it: https://github.com/Toptranslation/tt_cli/pull/11/files#diff-ecc4120ae09237ec53d471dc646014feL12